.github/workflows/ci.yml

```diff
@@ -51,14 +51,25 @@ jobs:
           --health-retries 5
     env:
       DATABASE_URL: postgres://postgres:postgres@localhost:5432/postgres
+      NPM_CONFIG_WORKSPACE: spark-api
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
           node-version: 20
       - run: npm ci
       - run: npm run migrate
-      - run: npm run test:publish
+      - run: npm test
+
+  lint-all:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - run: npm ci
+      - run: npm run lint
```

Comment:
I need to run `standard` for all files in the monorepo, including those that are not in any sub-package, e.g. `migrations/index.js`

Reply:
Could we turn `migrations` into a sub-package too?

Reply:
> Could we turn `migrations` into a sub-package too?

> I would prefer indeed if both would work: linting from the parent and the subpackage. Could we create a standard config that re-exports the parent config?

I am not a fan of this. It adds a lot of boilerplate. We will need to add package.json to `migrations`, add scripts like `lint`, add `standard` as a dependency, and so on.

Having written that, I also want to get this monorepo setup finished ASAP and choose the setup that works for both of us, so I am going to take the direction you suggested.

Reply:
> Could we turn migrations into a sub-package too?

Hmm, we would have to turn `bin` and `scripts` into sub-packages too. I definitely don't like that.

I am going to explore a different setup, where the monorepo root is a package on its own, and then some of the subdirectories are packages (workspaces) too.

Reply:
Done in https://github.com/filecoin-station/spark-api/pull/326/commits/daa26500505d69998ac67522d8b53eaa72ab5065

Reply:
I am keeping the `lint-all` step so that we can lint all workspaces in one go, and in parallel with running the tests.

The alternative is to lint `spark-api` and `spark-publish` as part of `build-*` workflows, and then add another workflow to run only files that are not part of any child workspace (`bin`, `migrations`, `scripts`). I am not sure how easy that would be to do, though, so I prefer the current proposal.